app/models/like.rb

```diff
@@ -0,0 +1,9 @@
+class Like < ApplicationRecord
+  belongs_to :user, foreign_key: 'AuthorId'
+  belongs_to :post, foreign_key: 'PostId'
```

Comment:
- [ ] Kindly apply thesame proposal I have made for the comment model here.
- `belongs_to :author, class_name: 'User'`
- `belongs_to :post`